.dagger/cli.go

```diff
@@ -61,8 +59,8 @@ func (cli *CLI) Publish(
 	}
 	ctr = ctr.
 		WithWorkdir("/app").
-		WithMountedDirectory("/app", cli.Dagger.Source()).
-		WithDirectory("/app/.git", gitDir)
+		WithDirectory("/app", cli.Dagger.Source()).
+		WithDirectory("/app", cli.Dagger.Git.Directory())
```

Comment:
Is `WithDirectory` additive, or does it change `/app` to `cli.Dagger.Git.Directory()` after the second call?

Reply:
It's additive, it uses `MergeOp` under the hood :tada: So we're merging the directories.